package.json

```diff
@@ -590,6 +597,10 @@
 					"command": "cosmosDB.copyConnectionString",
 					"when": "view == cosmosDBExplorer && viewItem == cosmosDBTableAccount"
 				},
+				{
+					"command": "cosmosDB.copyConnectionString",
+					"when": "view == cosmosDBExplorer && viewItem == mongoDb"
```

Comment:
Again - this is a UI change unrelated to the main PR and should be a separate for the reasons I mentioned earlier. I also forgot to mention this is important for the commit history (For example sometimes we look at the commit history when writing the changelog before a release)

Also you would need to change this `https://github.com/Microsoft/vscode-cosmosdb/blob/master/src/extension.ts#L112` to also accept `MongoDatabaseTreeItem`